packages/block-library/src/latest-posts/edit.js

```diff
@@ -20,19 +16,22 @@ import {
 	Spinner,
 	ToggleControl,
 	Toolbar,
+	RadioControl,
 } from '@wordpress/components';
 import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
 import { __ } from '@wordpress/i18n';
 import { dateI18n, format, __experimentalGetSettings } from '@wordpress/date';
+import { decodeEntities } from '@wordpress/html-entities';
 import {
 	InspectorControls,
 	BlockControls,
+	BlockAlignmentToolbar,
 } from '@wordpress/block-editor';
 import { withSelect } from '@wordpress/data';
 
 /**
- * Module Constants
+ *  Module Constants
```

Comment:
apparently there is an extra space here :)

Reply:
Haha don't even remember putting it in there